bids-validator/bids_validator/rules/file_level_rules.json

```diff
@@ -522,5 +522,37 @@
     "tokens": {
       "@@@_pet_ext_@@@": ["blood\\.tsv\\.gz", "blood\\.tsv", "blood\\.json"]
     }
+  },
+
+  "microscopy": {
```

Comment:
If we want to accept 1 json file per subject without the sample entity (as rat3 in data_axondeepseg_sem dataset), would it be possible to separate the regex rules for "images" extension and "json" extensions?
So "sample" would be required for images, but optional for json?

I saw a few modalities that have different rules for different files, for example "meg". Let me know what you think.